main.nf

```diff
@@ -298,9 +332,7 @@ if (params.gtf) {
     Channel
         .fromPath(params.gtf, checkIfExists: true)
         .ifEmpty { exit 1, "GTF annotation file not found: ${params.gtf}" }
-        .into { gtf_makeSTARindex; gtf_makeHisatSplicesites; gtf_makeHISATindex; gtf_makeSalmonIndex; gtf_makeBED12;
```

Comment:
@olgabot I am not sure if this change is correct - But I did it to not have the channels called by the operator here and in the later steps as outputs.